<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>testing</class>
 <widget class="QDialog" name="testing">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>451</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial Unicode MS</family>
    <pointsize>18</pointsize>
    <weight>75</weight>
    <bold>true</bold>
    <underline>false</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string>Live Updates</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Icons/car.png</normaloff>Icons/car.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#Qdialog(
background-color: rgb(154, 153, 150);)</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="target">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Target</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>SPEED:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="t_speed_2"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>STEERING</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="t_angle_2"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="vut">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>VUT</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>SPEED:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="v_speed"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>STEERING</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="v_angle"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>TIME</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="time_target"/>
     </item>
     <item>
      <widget class="QPushButton" name="stats">
       <property name="text">
        <string>Graph</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="data">
       <property name="text">
        <string>Stats</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="load_file">
       <property name="text">
        <string>LOAD</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Icons/load.png</normaloff>Icons/load.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="start">
       <property name="text">
        <string>START</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Icons/start.png</normaloff>Icons/start.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="replay">
       <property name="text">
        <string>REPLAY</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
